packages/preload/src/modules/workspace.ts

```diff
@@ -73,13 +71,14 @@ export async function getProject(_path: string): Promise<Project> {
 
     return {
       path: _path,
-      title: scene.display?.title,
+      title: scene.display?.title || 'Untitled scene',
       description: scene.display?.description,
       thumbnail: await getProjectThumbnail(_path, scene),
       layout: getRowsAndCols(parcels),
       createdAt: Number(stat.birthtime),
       updatedAt: Number(stat.mtime),
       size: stat.size,
+      scene,
```

Comment:
does it make sense to return title, description, etc if we are also returning the scene?

Reply:
I's not necessary for now, I'll remove it.